contrib/devtools/copyright_header.py

```diff
@@ -22,7 +22,6 @@
     'src/test/fuzz/FuzzedDataProvider.h',
     'src/tinyformat.h',
     'src/bench/nanobench.h',
-    'test/functional/test_framework/bignum.py',
```

Comment:
👍 was removed in https://github.com/bitcoin/bitcoin/commit/3ed772d2219e58d6afea3d12c0ebebb8487445e7